specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/iotSites.json

```diff
@@ -0,0 +1,261 @@
+          "Get IoT site": {
+            "$ref": "./examples/IotSites/Get.json"
+          }
+        },
+        "tags": [
+          "IoT Sites"
+        ],
+        "operationId": "IotSites_Get",
+        "description": "Get IoT site",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/IotSiteScope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IotSitesModel"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Create or update IoT site": {
+            "$ref": "./examples/IotSites/Put.json"
+          }
+        },
+        "tags": [
+          "IoT Sites"
+        ],
+        "operationId": "IotSites_CreateOrUpdate",
+        "description": "Create or update IoT site",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/IotSiteScope"
+          },
+          {
+            "$ref": "#/parameters/IotSitesModel"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated IoT site",
+            "schema": {
+              "$ref": "#/definitions/IotSitesModel"
+            }
+          },
+          "201": {
+            "description": "Created IoT site",
+            "schema": {
+              "$ref": "#/definitions/IotSitesModel"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete IoT site": {
+            "$ref": "./examples/IotSites/Delete.json"
+          }
+        },
+        "tags": [
+          "IoT Sites"
+        ],
+        "operationId": "IotSites_Delete",
+        "description": "Delete IoT site",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/IotSiteScope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted IoT site"
+          },
+          "204": {
+            "description": "IoT site does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
```

Comment:
Does this definition actually exist in that file? I can't seem to find in the `master` branch.

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/security/resource-manager/common/v1/types.json

Reply:
outdated, this file already merged into master